### DISCUSSION TITLE
Recovering read alignment from RPGG

### DISCUSSION THREAD

**Discussion by Pkaps25** [ChaissonLab/danbing-tk#9] [Q&A] [answered]
Hello,

Is there a way to see where in the RPGG reads from SRS samples align? Thank you.

**Comment by Pkaps25** [comment, marked as answer]
Resolved: Located the `-a` option for `danbing-tk align`

**Reply by joyeuxnoel8** [reply]
Yes, there it is! I just updated the documentation to include the output format.
